Python.Launcher.3.12 version 3.12.0

Dragon1573 opened the pull request:
Resolve https://github.com/microsoft/winget-pkgs/issues/121195 .

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?
- [x] Have you tested your manifest in Windows Sandbox with `Tools/SandboxTest.ps1 -Manifest <path>`?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121756)

Dragon1573 committed:
Python.Launcher.3.12 version 3.12.0

Dragon1573 commented:
@wingetbot waivers Add Validation-Executable-Error

zooba commented:
LGTM :shipit:

stephengillie commented:
@wingetbot waivers Add Validation-Executable-Error

mdanish-kh commented on manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+  Scope: user
+  InstallerUrl: https://www.python.org/ftp/python/3.12.0/win32/launcher.msi
+  InstallerSha256: 3A0DAA56286B02546438D24F158A334F4936B5EB027BECCDA60EAA0B39C075BA
+  InstallerSwitches:
+    Custom: ALLUSERS=0
```
```suggestion
```
Not required. The installer installs in user mode without these switches

mdanish-kh commented on manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+- pyz
+- pyzw
+ProductCode: '{3182A195-B671-44A8-B0C7-7876B916BA5A}'
+ElevationRequirement: elevatesSelf
```
```suggestion
```

The installer does not prompt for elevation if being installed in user mode.

mdanish-kh commented on manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.locale.en-US.yaml:
```diff
@@ -0,0 +1,35 @@
+- python3
+- script
+Documentations:
+- DocumentLabel: Documentation
+  DocumentUrl: https://www.python.org/doc/
```
```suggestion
- DocumentLabel: Python Launcher for Windows
  DocumentUrl: https://docs.python.org/3/using/windows.html#launcher
```

Dragon1573 committed:
Fix(Python.Launcher.3.12): Change moniker

- Fixup `Moniker` property of the manifests

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fix (Python.Launcher.3.12): Change short description

- Update "ShortDescription" property of the manifest
- Use official description

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Launcher.3.12): Update full description

- Update "Description" property to match official content.

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 commented on manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Python.Launcher.3.12
```
Okay, thank you! I'll change manifests directory on my local device.

Dragon1573 committed:
Fixup (Python.Launcher.3.12): Remove "InstallerSwitches"

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Launcher.3.12): Remove "ElevationRequirement"

The installer does not prompt for elevation if being installed in user mode.

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Launcher.3.12): Add "ElevationRequirement"

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Lancher.3.12): Add "AppsAndFeaturesEntries"

- DisplayVersion is needed to exactly match the version written to Registry.
- Rest of the metadata is generally useful for better matching.

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Launcher.3.12): Change documents info

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

Dragon1573 committed:
Fixup (Python.Launcher): Change "Package Identifier"

- Python.Launcher.3.12 ==> Python.Launcher

Dragon1573 committed:
Fixup (Python.Launcher): Rename manifest files

- Remove "3.12" suffix of each manifest file.

mdanish-kh reviewed (approved):
Since the path has been modified, this needs a waiver again to get merged

cc @stephengillie

stephengillie commented:
@wingetbot waivers Add Validation-Executable-Error
